examples/green-house-migration/legacy/greenhouse/batch/users.py

```
@@ -0,0 +1,37 @@
+
+    def fetch(self):
+        users = fetch_all_from_api("users")
+
+        for _user in users:
+            user_id = user.get("id")
+            if not user_id:
+                continue
+
+            try:
+                user["job_permissions"] = gh_get("users/{user_id}/permissions/jobs")
+            except UsersProcessorError as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)